docker/install.sh

```diff
@@ -9,10 +10,17 @@ if [ -e "jenkins_env.sh" ]; then
 	source "jenkins_env.sh"
 fi
 
+# Assumes docker
+docker pull mysql/mysql-server:5.7
+docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
+
 # catch sbt issues separately
 sbt update && echo "sbt updated successfully."
 
-sbt -J-Xms4g -J-Xmx4g test
+# Assumes docker-machine
+sbt -J-Xms4g -J-Xmx4g test -Ddockermysql.ip=`docker-machine ip default`
```

Comment:
likely fix in jenkins:rawls-dev-build. if successful, we will make change to all other rawls builds